css/features.css

```diff
@@ -0,0 +1,45 @@
+.features--pd-btm-0 {
```

Comment:
It would make more sense to move this class to a `section` block, as not only `Features` section might be without a bottom padding, just as `section--gray` can be applied to any section